content/updates/2020-06/3-manually-execute-naming-conventions-on-assets.md

```diff
@@ -0,0 +1,22 @@
+# Manually execute naming conventions on assets
+::: meta-data type="New" features="Asset Manager" available="late-June" link-to-doc="../articles/assets-product-link-rules.html#focus-on-the-naming-convention.html"
+:::
+
+In the Asset Manager, it is possible to define a naming convention in order to automatically populate asset attributes with values contained in the asset filename or asset code. This naming convention is executed at the asset creation.
+
+From now on, you can manually re-execute this naming convention afterward. It can be very useful if your naming convention changes after the asset upload/creation for instance.
```

Comment:
"This naming convention was executed at the asset creation but from now on, you can also manually re-execute this naming convention afterward!"